test/basic.html

```diff
@@ -25,11 +25,20 @@
 
   <script>
 
-    window.localStorage.setItem('core-localstorage-test', '{"foo":"bar"}');
-    var storage = document.querySelector('#localstorage');
-    
+    var storage;
+
     suite('basic', function() {
 
+      before(function() {
+        window.localStorage.setItem('core-localstorage-test', '{"foo":"bar"}');
+        storage = document.querySelector('#localstorage');
+        storage.load();
```

Comment:
I think this call to `load` may be redundant. I would expect `nameChanged` to always be called, and therefore I expect `load` to always be called, when the element is configured.